Assets/Scripts/Screeps3D/RoomObjects/Views/CreepActionView.cs

```diff
@@ -156,9 +155,12 @@ public void Unload(RoomObject roomObject)
         private void doAura(string auraType, Color32 auraColor) {
             var target = _creep.Actions[auraType];
             switch(auraType) {
+                case "rangedAttack":
```

Comment:
A little confused, is rangedAttack an aura?

Reply:
it's more like it triggers an aura-like effect

Reply:
it still triggers beam effect